pkg/login/social/generic_oauth.go

```diff
@@ -24,6 +24,8 @@ type SocialGenericOAuth struct {
 	emailAttributeName   string
 	emailAttributePath   string
 	loginAttributePath   string
+	nameAttributeName    string
```

Comment:
Why do we add both `nameAttributeName` and `nameAttributePath`? Isn't the `name` variant of other properties just a legacy setting from before we supported JMESPaths?

Reply:
Hm, good question, I'll check it out. Looks like for login we only have `loginAttributePath` option.

Reply:
@marefr What do you think?